toolkit/tools/imagegen/configuration/configuration.go

```diff
@@ -38,6 +38,13 @@ type TargetDisk struct {
 	Value string `json:"Value"`
 }
 
+// PreInstallScript defines a script to be run before installation
+// starts and provides a way to pass parameters to it.
+type PreInstallScript struct {
```

Comment:
Since it's identical to `PostInstallScript` let's avoid repetition and use the same type for both (we could re-name `PostInstallScript` to something like `InstallScript` or `InstallationScript`, etc.).

Reply:
Fixed